chain/filecoin/gas.go

```diff
@@ -51,7 +50,7 @@ func (gasEstimator *GasEstimator) EstimateGas(ctx context.Context) (pack.U256, p
 	}
 
 	// Estimate the gas fee cap and gas premium fields for this dummy message.
-	msgOut, err := gasEstimator.client.node.GasEstimateMessageGas(ctx, &msgIn, &api.DefaultMessageSendSpec, types.EmptyTSK)
+	msgOut, err := gasEstimator.client.node.GasEstimateMessageGas(ctx, &msgIn, nil, types.EmptyTSK)
```

Comment:
What is this change for?

Reply:
Aftr having upgraded lotus to v1.9.0, their `api` package no longer has a `DefaultMessageSendSpec`. Instead a `nil` should be used.

Reply:
Too easy.